### ISSUE TITLE
pool.info command appears broken

### ISSUE THREAD

**Issue by Symmetric** [vmware/govmomi#203]
Running the pool.info command just returns the usage text, with no error message:

```
$ govc pool.info
Usage: govc pool.info [OPTIONS]

  -dc=Datacenter                       Datacenter [GOVC_DATACENTER]
  -debug=false                            Store debug logs [GOVC_DEBUG]
  -json=false                             Enable JSON output
  -k=true                                 Skip verification of server certificate [GOVC_INSECURE]
  -u=https://vcenter/sdk          ESX or vCenter URL [GOVC_URL]
```

Here's the contents of the debug directory:

```
$ cat *
POST /sdk HTTP/1.1
Host: vcenter
Content-Type: text/xml; charset="utf-8"
Soapaction: urn:vim25/5.5

<?xml version="1.0" encoding="UTF-8"?>
<Envelope xmlns="http://schemas.xmlsoap.org/soap/envelope/"><Body><RetrieveProperties xmlns="urn:vim25"><_this type="PropertyCollector">propertyCollector</_this><specSet><propSet><type>SessionManager</type><pathSet>currentSession</pathSet></propSet><objectSet><obj type="SessionManager">SessionManager</obj></objectSet></specSet></RetrieveProperties></Body></Envelope>HTTP/1.1 200 OK
Content-Length: 957
Cache-Control: no-cache
Connection: Keep-Alive
Content-Type: text/xml; charset=utf-8
Date: Mon, 19 Jan 2015 22:49:30 GMT

<?xml version="1.0" encoding="UTF-8"?>
<soapenv:Envelope xmlns:soapenc="http://schemas.xmlsoap.org/soap/encoding/"
 xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/"
 xmlns:xsd="http://www.w3.org/2001/XMLSchema"
 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
<soapenv:Body>
<RetrievePropertiesResponse xmlns="urn:vim25"><returnval><obj type="SessionManager">SessionManager</obj><propSet><name>currentSession</name><val xsi:type="UserSession"><key>5238f2f9-a08f-f314-7197-480f3f306b2c</key><userName>_snip_</userName><fullName>_snip_</fullName><loginTime>2015-01-19T22:10:25.078313Z</loginTime><lastActiveTime>2015-01-19T22:46:10.307388Z</lastActiveTime><locale>en</locale><messageLocale>en</messageLocale><extensionSession>false</extensionSession><ipAddress>_snip_</ipAddress><userAgent>Go 1.1 package http</userAgent><callCount>4</callCount></val></propSet></returnval></RetrievePropertiesResponse>
</soapenv:Body>
</soapenv:Envelope>2015-01-19T14-49-31.119709708:    1 took     95ms
```

Looking at the [code](https://github.com/vmware/govmomi/blob/master/govc/pool/info.go#L50), it appears that the command is expecting an argument, so it looks like (at least) the usage statement is incorrect.

When I try passing in a valid resource pool for my datacenter, the command returns a success RC, but no output at all.

**Comment by dougm** [comment]
Similar to issue #190 ... the argument should be an inventory path relative to the datacenter hostFolder, for example '*/Resources' or the full path:

govc ls -l 'host/*' | grep ResourcePool | awk '{print $1}' | xargs -n1 -t govc pool.info

**Comment by pietern** [comment]
This is a documentation issue, fixed by #210.

The help output is now as follows:
```
$ ./govc pool.info -help
Usage: ./govc pool.info [OPTIONS] POOL...
Retrieve information about one or more resource POOLs.

POOL may be an absolute or relative path to a resource pool or a (clustered)
compute host. If it resolves to a compute host, the associated root resource
pool is returned. If a relative path is specified, it is resolved with respect
to the current datacenter's "host" folder (i.e. /ha-datacenter/host).

Paths to nested resource pools must traverse through the root resource pool of
the selected compute host, i.e. "compute-host/Resources/nested-pool".

The same globbing rules that apply to the "ls" command apply here. For example,
POOL may be specified as "*/Resources/*" to expand to all resource pools that
are nested one level under the root resource pool, on all (clustered) compute
hosts in the current datacenter.

Options:
  -dc=                            Datacenter [GOVC_DATACENTER]
  -debug=false                    Store debug logs [GOVC_DEBUG]
  -json=false                     Enable JSON output
  -k=true                         Skip verification of server certificate [GOVC_INSECURE]
  -u=https://127.0.0.1:18443/sdk  ESX or vCenter URL [GOVC_URL]
```
